src/forms/create_ticket.ts

```diff
@@ -64,17 +68,17 @@ class FormFields extends BaseFormFields {
 
     // setState sets state for the current Zendesk Form
     async setState(): Promise<void> {
-        const listReq = this.zdClient.ticketforms.list();
+        const listReq = this.zdClient?.ticketforms.list();
         const zdTicketForms = await tryPromiseWithMessage(listReq, 'Failed to fetch ticket forms');
         this.zdTicketForms = zdTicketForms;
     }
 
     async addFormSelectDependentFields(): Promise<void> {
         // get the form id from the selected form field value
-        const formID = this.builder.getFieldValueByName(CreateTicketFields.NameFormsSelect);
+        const formID = this.builder.getFieldValueByName(CreateTicketFields.NameFormsSelect) as unknown;
 
-        const zdFormFieldIDs = this.getTicketFieldIDs(formID);
-        const fieldsListReq = this.zdClient.ticketfields.list();
+        const zdFormFieldIDs = this.getTicketFieldIDs(formID as number);
```

Comment:
Why not use `ToString` instead of two type assertions?

Reply:
remove the `as unknown` and now casting formID as a string.